configure.ac

```diff
@@ -215,7 +215,7 @@ fi
 # extension, which is compiled-out of most platform sqlites. We therefore
 # always use our own bundled copy, version 3.26.0 at the time of this
 # writing.
-sqlite3_CFLAGS='-isystem $(top_srcdir)/lib/sqlite -DSQLITE_CORE'
+sqlite3_CFLAGS='-isystem $(top_srcdir)/lib/sqlite -DSQLITE_CORE -DSQLITE_OMIT_LOAD_EXTENSION=1'
```

Comment:
Nitpick: I think it would be better if this change (re-adding this define) was squashed into the first commit in this PR.

Reply:
ok

Reply:
fixed